src/Browscap/Filter/LiteFilter.php

```diff
@@ -83,7 +79,7 @@ public function isOutputSection(array $section)
      *
      * @return bool
      */
-    public function isOutputProperty($property, ?WriterInterface $writer = null)
+    public function isOutputProperty(string $property, ?WriterInterface $writer = null) : bool
```

Comment:
Ideally squash out optional deps; if it's not "required" just create a `DummyWriter` which does nothing for example